Cogs.Tests/Cogs.Tests.csproj

```diff
@@ -7,6 +7,18 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Remove="cogsburger.zip" />
+    <None Remove="dotnet.exe" />
+    <None Remove="normativeXMI.xsd" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="cogsburger.zip" />

```

Comment:
Zipping and unzipping cogsburger seems unnecessary to me (I don't know why I did it, to begin with), so instead, I just got the path to it from the testers using path.combine